enforcer/model.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Optional, Dict, Any, List
+
+from pydantic import BaseModel
+
+
+class PodOwner(BaseModel):
+    kind: str
+    name: str
+    namespace: str
+
+class RsOwner(BaseModel):
+    rs_name: str
+    namespace: str
+    owner_name: str
+    owner_kind: str
+    deletion_ts: Optional[float] = None
+
+class Resources(BaseModel):
+    request: float
+    limit: Optional[float]
+
+
+class ResourcesRecommendation(BaseModel):
+    cpu: Optional[Resources] = None
+    memory: Optional[Resources] = None
+
+class WorkloadRecommendation(BaseModel):
+    workload_key: str
+    container_recommendations: Dict[str, ResourcesRecommendation] = {}
+
+    def get(self, container: str) -> Optional[ResourcesRecommendation]:
+        return self.container_recommendations.get(container, None)
+
+    @staticmethod
+    def build(recommendation: Dict[str, Any]) -> Optional[ResourcesRecommendation]:
```

Comment:
I suggest moving this to the ResourcesRecommendation class. This function confused me for a while as I assumed it was building a WorkloadRecommendation and I missed the return type annotation.

Reply:
fixed